catalog/ui/catalog-ui-search/src/main/webapp/component/location-old/location-old.view.js

```diff
@@ -113,7 +113,7 @@ define([
                         if (CQLUtils.isPointRadiusFilter(filter)){
                             var pointText = filter.value.value.substring(6);
                             pointText = pointText.substring(0, pointText.length - 1);
-                            latLon = pointText.split(' ');
+                            var latLon = pointText.split(' ');
```

Comment:
If you have babel and you want block scope, you might want to consider using `let`.